### ISSUE TITLE
`AuthorizedKeysCommand` will fail if running with SELinux mode set to `enforcing`

### ISSUE THREAD

**Issue by jmorganca** [infrahq/infra#3893]
When running on Linux with SELinux enabled and set to `enforcing` mode (e.g. an out-of-the-box RHEL server on AWS), outbound requests from `AuthorizedKeysCommand` will fail with `permission denied`. This seems to stem from an SELinux policy around `AuthorizedKeysCommand`.

**Comment by dnephin** [comment]
Good find! Red hat has docs of their own that suggest using this command ([ref](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/linux_domain_identity_authentication_and_policy_guide/openssh-sssd#openssh-sssd-users)), so there must be some way to make it work. 

Did you use `AuthorizedKeysCommandRunAs nobody` ?

Do you have the logs from `sshd` ? There should be some indication in those logs of which policy needs to be changed to allow this.  I tried to google for it, but I have not found anything interesting yet.

**Comment by jmorganca** [comment]
@dnephin I did! Both `AuthorizedKeysCommandRunAs nobody` and `AuthorizedKeysCommandRunAs root` (for testing) seem to have the same issue.

`sshd` logs:

```
Dec 19 21:36:52 ip-172-31-27-251.us-east-2.compute.internal infra-ssh[4452]: {"level":"error","error":"api list users: GET \"/api/users\": Get \"https://<redacted>/api/users?group=&limit=0&name=&page=0&publicKeyFingerprint=SHA256%3AFAoY77Xr0rv32xMvNlb7JWkgSkHc2NBXBqaS9SHiNXY&showSystem=false\": dial tcp 44.228.17.134:443: connect: permission denied","message":"ssh auth-keys exit"}
Dec 19 21:36:52 ip-172-31-27-251.us-east-2.compute.internal sshd[4450]: AuthorizedKeysCommand /usr/local/sbin/infra sshd auth-keys jeff SHA256:FAoY77Xr0rv32xMvNlb7JWkgSkHc2NBXBqaS9SHiNXY failed, status 1
Dec 19 21:36:52 ip-172-31-27-251.us-east-2.compute.internal sshd[4450]: Connection closed by authenticating user jeff 38.88.100.203 port 53078 [preauth]
```
